app/models/booking.rb

```diff
@@ -0,0 +1,21 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :listing
+
+  STATUS = ["Accepted", "Pending", "Cancelled"]
+
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validate :end_date_is_after_start_date
+  validates :status, presence: true, inclusion: { in: STATUS, message: "%{value} is not a valid status" }
```

Comment:
Do we need the message: "%{value} is not a valid status" } ? Are we not asking them to chose from listed three choices only?

Reply:
That makes sense. If we implement validation from user input then the message is probably not necessary. Let me remove it.